app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,3 @@
+class HotelsController < ApplicationController
+  
+end
```

Comment:
Looks like you were on your way to try and tackle the extension before you had the 3 user stories complete. I would hold off on the extension until all your user stories are complete.